components/shared/embedder/lib.rs

```diff
@@ -711,7 +711,7 @@ pub struct ScreenGeometry {
     pub size: DeviceIntSize,
     /// The available size of the screen in device pixels. This size is the size
     /// available for web content on the screen, and should be `size` minus any system
-    /// toolbars, docks, and interface elements of the browser. This will be converted to
+    /// toolbars, docks, and interface elements of the OS. This will be converted to
```

Comment:
"system" is already used at the start of the sentence so you can omit "of the OS" here.

```suggestion
    /// toolbars, docks, and interface elements. This will be converted to
```